instructure/canvas-ios
# Change letter grade when toggling based on graded assignments on grade screen

refs: [MBL-17951](https://instructure.atlassian.net/browse/MBL-17951)
affects: Student , Parent
release note: Added letter grade when toggling based on graded assignments on grade screen
### Test plan:
- Open student or parent app.
- Navigate to grade.
- Change toggle graded assignments state  will see the grade letter change.
## Screenshots

https://github.com/user-attachments/assets/3acb0e31-ef04-4e5b-bb6f-b157dbd14bd4


https://github.com/user-attachments/assets/d211c7d1-ad19-47f9-82d2-e937f607b56a


## Checklist


[MBL-17951]: https://instructure.atlassian.net/browse/MBL-17951?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
